src/test.zig

```diff
@@ -154,6 +140,261 @@ const ErrorMsg = union(enum) {
     }
 };
 
+/// Default config values for known test manifest key-value pairings.
```

Comment:
"known test manifest" is unclear.
What does this mean? Why can the target triplet not being reused etc?

Reply:
What do you mean by a target triplet?

Reply:
I thought there might be a target triplet parser, which could be reused. However, currently I dont find one.

Reply:
Oh, but we have one and it's used here by default. Have a look at line #385 of the file:

```zig
            .Struct => if (comptime std.mem.eql(u8, @typeName(T), "CrossTarget")) return struct {
                fn parse(str: []const u8) ?T {
                    var opts = CrossTarget.ParseOptions{
                        .arch_os_abi = str,
                    };
                    return CrossTarget.parse(opts) catch null;
                }
            }.parse else @compileError("no default parser for " ++ @typeName(T)),
```

The reason `TestManifestConfigDefaults` returns triplets as a string is to feed uniformly into the config iterator and parsing function. This seemed like the cleanest and simplest way of the iterator *always* returning a stream for each required config value such `backend` and `target`.